include/srf/benchmarking/segment_watcher.hpp

```diff
@@ -22,6 +22,8 @@
 #include <srf/core/executor.hpp>
 
 #include <glog/logging.h>
```

Comment:
Should this be in sorted order?

Reply:
Interesting. Looks like our .clang-tidy was prioritizing .h files over .hpp. Updated the .clang-tidy and ran it over all files to match.